slices/gearman-tools.yaml

```diff
@@ -0,0 +1,22 @@
+package: gearman-tools
+
+essential:
+  - gearman-tools_copyright
+
+slices:
+  bins:
+    contents: 
+      /usr/bin/gearadmin:
+      /usr/bin/gearman:
+
+  libs:
+    essential:
+      - libgearman8t64_libs
+      - libboost-program-options1.83.0_libs
+      - libc6_libs
+      - libgcc-s1_libs
+      - libstdc++6_libs
```

Comment:
There's no need for a `libs` slice for gearman-tools.
Instead you can put the essential part under the `bins` slice, like this:
```
  bins:
    essential:
      - libgearman8t64_libs
      - libboost-program-options1.83.0_libs
      - libc6_libs
      - libgcc-s1_libs
      - libstdc++6_libs
    contents: 
      /usr/bin/gearadmin:
      /usr/bin/gearman:
```